engine/engine_test.go

```diff
@@ -165,8 +166,15 @@ func TestStartStopTwoDoesNotCausePanic(t *testing.T) {
 }
 
 func TestCheckExchangeExists(t *testing.T) {
-	e := CreateTestBot(t)
-
+	t.Parallel()
+	em := SetupExchangeManager()
+	exch, err := em.NewExchangeByName(testExchange)
+	if !errors.Is(err, nil) {
+		t.Errorf("received '%v' expected '%v'", err, nil)
```

Comment:
We should t.Fatal here otherwise the next line `exch.SetDefaults()` will throw a nil pointer error as NewExchangeByName will return a nil IBotInterface. This would occur for all the similar tests below.